packages/components/webpack.config.js

```diff
@@ -1,25 +1,25 @@
-const StyleLintPlugin           = require('stylelint-webpack-plugin');
-const SpriteLoaderPlugin        = require('svg-sprite-loader/plugin');
-const MiniCssExtractPlugin      = require("mini-css-extract-plugin");
-const path                      = require('path');
+const StyleLintPlugin      = require('stylelint-webpack-plugin');
+const SpriteLoaderPlugin   = require('svg-sprite-loader/plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path                 = require('path');
 
 const isServe = process.env.BUILD_MODE === 'serve';
 
 module.exports = {
     // entry: path.join(__dirname, 'src', 'index.js'),
     entry: {
         // index: path.join(__dirname, 'src', 'index.js'),
-        button: path.resolve(__dirname, 'src' ,'components/button/index.js'),
-        label: path.resolve(__dirname, 'src' , 'components/label/index.js'),
+        button: path.resolve(__dirname, 'src', 'components/button/index.js'),
+        label: path.resolve(__dirname, 'src', 'components/label/index.js'),
     },
     output: {
         path: path.resolve(__dirname, 'lib'),
-        filename: "[name].js",
+        filename: '[name].js',
         libraryExport: 'default',
-        library: ["deriv-component", "[name]"],
+        library: ['deriv-component', '[name]'],
```

Comment:
This is not working for multiple `scss` resources. when adding a new component with `scss` imported, it throws the error: 

```sh
ERROR in chunk [COMPONENT_NAME] [entry]
deriv-components.css
Conflict: Multiple chunks emit assets to the same filename deriv-components.css (chunks 0 and 1)
```

Reply:
@reza-fs this PR is fixing this issue. now we are importing every css file separatly. 
so if you want button you do 

> import Button from 'deriv-components/lib/Button';
> import 'deriv-components/lib/button.css';

or you used `deriv-component-loader` which will handle this for you and you can import

`import {button, label,....} from 'deriv-componetns` and it do the same:
> import Button from 'deriv-components/lib/Button';
> import 'deriv-components/lib/button.css';
> import Label from 'deriv-components/lib/label';
> import 'deriv-components/lib/label.css';